apps/web/test/lib/getSchedule.test.ts

```diff
@@ -1526,21 +1526,20 @@ describe("getSchedule", () => {
       });
 
       const scenario = await createBookingScenario(
-        getScenarioData({
-          eventTypes: [
-            {
-              id: 1,
-              slotInterval: 45,
-              length: 45,
-              users: [
-                {
-                  id: 101,
-                },
-              ],
-            },
-          ],
-          organizer,
-        })
+        getScenarioData(
+          {
+            eventTypes: [
+              {
+                id: 1,
+                slotInterval: 45,
+                length: 45,
+                userId: 101,
```

Comment:
It's an individual event inside an org, so we don't check with users, but userId. No idea how this was passing in the first place haha